pkg/handlers/template.go

```diff
@@ -0,0 +1,146 @@
+package handlers
+
+import (
+	"bytes"
+	"fmt"
+	"io/fs"
+	"log"
```

Comment:
nit - noticed we're using log here vs logrus in the config package - is this intentional/should we standardize?

Reply:
nice catch, ill make the change